### PR TITLE
feat: add LICENSE file and link to readme

### PR DESCRIPTION
**Pull request by CsanaadB**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Purpose Industries
```
Comment (ruffle1986) [inline LICENSE]: it's 2016

File: readme.markdown [hunk 2]
```diff
@@ -90,3 +90,8 @@ Example:
 To develop run `npm start`.
 You can test your changes by opening `localhost:4567` in a browser.
 When you are happy with your changes you can build to dist with `npm run build`.
+
+
+## License
```
Comment (ruffle1986) [inline readme.markdown:95]: What do you think about following this?:

```markdown
## License

[MIT](LICENSE) &copy; Purpose Industries
```

I'd just like to show the licence directly in the readme.

File: LICENSE (hunk 1 above)
Comment (CsanaadB) [reply LICENSE]: lol thanks.. and it's almost done already

File: readme.markdown (hunk 2 above)
Comment (CsanaadB) [reply readme.markdown:95]: sure

**Comment by ruffle1986** [comment]
🍻

**Pull request merged**